ninja_ide/tools/utils.py

```diff
@@ -25,15 +26,69 @@ def get_home_dir():
 
 def get_python():
     found = []
-    for search_path in ('/usr/bin', '/usr/local/bin'):
-        files = os.listdir(search_path)
-        for fname in files:
-            if fname.startswith('python') and not fname.count('config') \
-                    and not fname.endswith("m"):
-                found.append(os.path.join(search_path, fname))
+    cwd = os.getcwd()
+
+    # search current folder first
+    path = search_folder('^python.exe$', cwd)
+
+    if path != '':
+        found.append(path)
+
+    else:
+        if sys.platform == 'win32':
```

Comment:
Please use the variables: **IS_WINDOWS**, **IS_MACOS** located in `ninja_ide.code.settings`